### ISSUE TITLE
asset generation only in production mode?

### ISSUE THREAD

**Issue by mhsdesign** [patriceckhart/NeosRulez.DynamicResources#1]
I tried out your great boilerplate: https://github.com/patriceckhart/NeosCMS-Boilerplate

but the assets (css/js) are only created when using production mode

```
FLOW_CONTEXT=Production ./flow server:run
```

a simple `./flow server:run` or `FLOW_CONTEXT=Development./flow server:run` would lead to this error on the asset url:
```
Notice: Undefined index: FLOW_CONTEXT in /NeosCMS-Boilerplate/Data/Temporary/Development/Cache/Code/Flow_Object_Classes/NeosRulez_DynamicResources_Controller_ResourceController.php line 57
```

Is this wanted behavior?
If so, I would make a pullrequest and add this info to the Readme for the NeosCMS-Boilerplate ;)

**Comment by patriceckhart** [comment]
I've never used `./flow server:run`. Environment variable FLOW_CONTEXT isn't set? 

It is not wanted. The resources must always be generated. They are just not cached in Development mode.

But to avoid this I will add a condition. Then it will work.

**Comment by patriceckhart** [comment]
fixed in 1.0.3

**Comment by mhsdesign** [comment]
Yeah it works now ;)
Thanks ^^
